<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorPositionWidget</class>
 <widget class="QWidget" name="InspectorPositionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>91</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,1,0,1,0,1">
     <item>
      <widget class="QLabel" name="xLabel">
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="xSpin">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>23</height>
        </size>
       </property>
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="yLabel">
       <property name="text">
        <string>y</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="ySpin">
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="zLabel">
       <property name="text">
        <string>z</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="zSpin">
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="wLabel">
       <property name="text">
        <string>w</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="wSpin">
       <property name="decimals">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="appBtn">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
